### PR TITLE
feat(artifacts): Add support for GitHub artifact account

### PR DESCRIPTION
**Pull request by wjoel**
Allows Halyard to configure clouddriver to download artifacts from GitHub, see https://github.com/spinnaker/clouddriver/pull/2231

### CODE REVIEW

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubEditArtifactAccountCommand.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright 2017 Google, Inc.
```
Comment (lwander) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubEditArtifactAccountCommand.java]: You can drop Google from all of these

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubAddArtifactAccountCommand.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  )
+  private String password;
+  @Parameter(
+      names = "--username-password-file",
+      converter = PathExpandingConverter.class,
+      description = "File containing \"username:password\" to use for GitHub authentication"
+  )
+  private String usernamePasswordFile;
+  @Parameter(
+      names = "--token",
```
Comment (wjoel) [inline halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubAddArtifactAccountCommand.java]: @lwander should this also have `password = true`?

Comment (lwander) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubAddArtifactAccountCommand.java]: ah good catch, yes

Comment (wjoel) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubAddArtifactAccountCommand.java]: done

File: halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubEditArtifactAccountCommand.java (hunk 1 above)
Comment (wjoel) [reply halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/github/GitHubEditArtifactAccountCommand.java]: done

**Pull request merged**